set.mm

```diff
@@ -500009,141 +500235,250 @@ The sumset operation can be used for both group (additive) operations and
     elrspunidl.1 $e |- .0. = ( 0g ` R ) $.
     elrspunidl.x $e |- .x. = ( .r ` R ) $.
     elrspunidl.r $e |- ( ph -> R e. Ring ) $.
-    elrspunidl.i $e |- ( ph -> S C_ ( LIdeal ` R ) ) $.
-    $( Elementhood to the span of a union of ideals.  (Contributed by Thierry
-       Arnoux, 30-Jun-2024.) $)
-    elrspunidl $p |- ( ph -> ( X e. ( N ` U. S ) <-> E. a e. ( B ^m S )
+    ${
+      elrspunidl.i $e |- ( ph -> S C_ ( LIdeal ` R ) ) $.
+      $( Elementhood to the span of a union of ideals.  (Contributed by Thierry
```

Comment:
"Elementhood in ..."

Reply:
Done in https://github.com/metamath/set.mm/pull/4702/commits/cba959dc98515e7a4ee85cdfab98c8e7eb598b54